### DISCUSSION TITLE
Useful But Huge

### DISCUSSION THREAD

**Discussion by mirshahreza** [FranckFreiburger/vue3-sfc-loader#193] [Ideas]
It is a good solution for my scenario but it is very huge :( it is about 1.8 M but your old version for vue2.x is very small.
Is it possible to have a light version? what is the functionality of this version that is so huge?
Is it possible to split functionalities to separated files?
My problem is size of your magic library.

**Comment by FranckFreiburger** [comment]
Hi,

https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/faq.md#why-vue3-sfc-loader-is-so-big

At the moment, it is not possible to create a "light" version or split anything of this library.

If size is an issue, try to use [vitejs ](https://vitejs.dev/) instead.
